Sources/KlaviyoCore/Models/APIModels/FullForms.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FullForms.swift
+//  klaviyo-swift-sdk
+//
+//  Created by Andrew Balmer on 11/25/24.
+//
+
+import Foundation
+
+public struct FullForms: Equatable {
```

Comment:
We will probably just be injecting this into javascript but for identifying the trigger I was wondering if having types is easier.

Reply:
sorry, just to be clear, when you say

> if we know the fields we care about, would it make sense to code them into a data type instead trying to get them from a dictionary?

do you mean "if we know the fields we care about, would it make sense to code them into a `Data` type instead trying to get them from a dictionary?"

or do you mean "if we know the fields we care about, would it make sense to code them into a [concrete Swift] data type [i.e., a Struct] instead trying to get them from a dictionary?"

if the former, that's what I'm already doing, right?

if the latter, I was just trying to avoid decoding the entire JSON response into a Swift object because, based on my conversations with Dan, we don't actually need to do anything with the JSON other than feed it into the Javascript for the Webview. So if we're decoding the response into a Swift type then we'd simply be re-encoding it in order to pass it to the Javascript. The only reason I'm doing any decoding at all (into the `FullForms` type) is because we need to extract the first item in the `full_forms` array. Otherwise I'd simply fetch the `Data` and not bother decoding any of it

Reply:
yeah I was referring to the later. I'm not suggesting we decode the entire json data into a struct but may be just the fields we care about. There is a thread somewhere on slack about this and I'll check back there and tag you. Of course we don't need to do this as part of this PR since it's not in scope but just letting you know.

Reply:
Addressed this; see my [comment below](https://github.com/klaviyo/klaviyo-swift-sdk/pull/240#discussion_r1861345994)